### DISCUSSION TITLE
如何修改密碼長度限制

### DISCUSSION THREAD

**Discussion by dearsiang** [hhyo/Archery#2801] [General]
Hi 大大

因為資安需求，需要Archery 建置帳號的密碼長度，需要12碼以上，但這邊建置Archery 只有顯示9碼，請問如何修改？

![Snipaste_2024-09-11_15-04-46](https://github.com/user-attachments/assets/92e6adc4-cde3-4f55-90cf-7e39a84b2b0c)

**Comment by LeoQuote** [comment]
你好, 这里的内容是可以通过修改 django 配置来实现的, https://docs.djangoproject.com/en/5.1/topics/auth/passwords/#enabling-password-validation 请参考这里的文档, 如有疑问可以咨询 django 社区

**Comment by dearsiang** [comment]
Hi Leo 大大

謝謝回覆

想請問，這邊使用docker 容器裝起來，是登入到容器裡面容器裡面，修改哪一個路徑的配置檔呢？
